WordPress/WordPressTest/LikeUserHelperTests.swift

```diff
@@ -3,15 +3,21 @@ import XCTest
 
 class LikeUserHelperTests: CoreDataTestCase {
 
-    func createTestRemoteUserDictionary(withPreferredBlog hasPreferredBlog: Bool) -> [String: Any] {
+    var siteID: NSNumber = 20
+
+    func createTestRemoteUserDictionary(
+        withPreferredBlog hasPreferredBlog: Bool,
+        siteID: Int? = nil,
+        year: Int = 2021
+    ) -> [String: Any] {
         var remoteUserDictionary: [String: Any] = [
-            "ID": 15,
+            "ID": Int.random(in: 0...Int.max),
```

Comment:
I can see why this uses random: So that we are almost guaranteed different unique IDs without having to manually set them.

However, randomness comes with the disadvantage of losing some determinism on the tests, given that each run will now almost certainly use different input data.

In the context of this PR, I think it's more important to merge the bug fix than to address this edge concern. I might open a followup if I have time later.